### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Arinpe**
This is part of the Microverse curriculum. This app is used to know the weather situation of a place. I had an initial repository before this but I had to let it go because it was giving me some issues

### CODE REVIEW

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Invalid PR 😞 😞 😞 
You have failing linter cases and as such I am marking this PR as invalid. Fix all failing cases and send for another review.


### Martin Afani
#### Slack: whiz25

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @Arinpe**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] Make sure you add error handling to the async/await function and display an error message when a user inputs an invalid city name. 

 

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Review by MahmoudBakr23 — COMMENTED** [review]
# Hi @Arinpe😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

## **Invalid Submission**

- Kindly fix the linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  //   document.querySelector('.temp').innerText = temp + unitSymbol;
+  //   document.querySelector(
+  //     '.humidity',
+  //   ).innerText = `Humidity: ${humidity}%  `;
+  //   document.querySelector(
+  //     '.wind',
+  //   ).innerText = `Wind speed ${speed}${windSpeed}`;
+  //   document.querySelector('.weather').classList.remove('Loading');
+  //   document.body.style.backgroundImage = `url('http://source.unsplash.com/1600x900/?${name} ')`;
+  // });
```
Comment (MahmoudBakr23) [inline src/index.js]: - [x] Kindly make sure to delete all the commented-out code from your files for more optimized and cleaner code 🙏

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          '.humidity',
+        ).innerText = `Humidity: ${humidity}%`;
+        document.querySelector(
+          '.wind',
+        ).innerText = `Wind speed ${speed}${windSpeed}`;
+        document.querySelector('.weather').classList.remove('Loading');
+        document.body.style.backgroundImage = `url('http://source.unsplash.com/1600x900/?${name} ')`;
+      });
+  } catch (e) {
+    document.querySelector('city').innerText = '';
```
Comment (MahmoudBakr23) [inline src/index.js]: - [ ] As the previous reviewer mentioned, kindly add an error handling when the user inserts a wrong entry.
error_handling_requirement

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @Arinpe 😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-shammar — APPROVED** [review]
Hi @Arinpe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**